source/developers/add_new_platform.markdown

```diff
@@ -61,3 +61,29 @@ The Discovery component is capable of setting up your components before firing t
 <p class='note warning'>
 This option is currently limited to built-in components.
 </p>
+
+### {% linkable_title Creating Entities %}
+
+Home Assistant will call a function with the following signature to initialize
```

Comment:
It looks like you're mixing up a component and a platform in this section. For a platform that is part of a component that uses the EntityComponent helper class, it is required for the parameter passed to `add_devices` to be an iterable (ie List) of objects that inherit the Entity class.